controller/api/public/stat_summary.go

```diff
@@ -340,7 +408,33 @@ func (s *grpcServer) getStatMetrics(ctx context.Context, req *pb.StatSummaryRequ
 		promQueries[promTCPReadBytes] = tcpReadBytesQuery
 		promQueries[promTCPWriteBytes] = tcpWriteBytesQuery
 	}
-	results, err := s.getPrometheusMetrics(ctx, promQueries, latencyQuantileQuery, reqLabels.String(), timeWindow, groupBy.String())
+
+	stringifiedReqLabels := reqLabels.String()
+	stringifiedGroupBy := groupBy.String()
+
+	if req.Selector.Resource.Type == tsString {
+
+		re := regexp.MustCompile("trafficsplit")
+		stringifiedGroupBy = re.ReplaceAllString(stringifiedGroupBy, "authority")
+	}
+	var results []promResult
+	var err error
+
+	if req.Selector.Resource.Type == tsString {
+		for key := range k8sObjects {
+			// use regex to look for any authority that matches the leaf service name -- this needs to be thought out since could return
+			// false positives
+			stringifiedReqLabels = `{direction="inbound", namespace="default", authority=~"^` + key.Leaf + `[.].+"}`
```

Comment:
Re: "outbound" vs. "inbound", we had discussed concerns that looking at "outbound" metrics means missing traffic that comes from outside the mesh, whereas "inbound" metrics would capture all traffic to each service within the trafficsplit (but those metrics don't have a `dst_service` so matching them to the leaf service is more complex), and we never landed on a clear answer.

I'm fine with measuring with "outbound" metrics but want to explicitly call out the tradeoff we're making. cc @grampelberg

Reply:
By focusing just on TrafficSplit, outbound makes sense because a split can only every happen on the outbound. This is one of the big reasons why making something specific to splits is much smaller scope than doing a general service view.

Reply:
Agreed that the split is only happening on the outbound. However if an unmeshed resource is sending requests to apex service `foo` in my trafficsplit `ts-foo`, the fact that 20% of traffic was split to leaf service `foo-1` and 80% of traffic was split to leaf service `foo-2` won't be captured since we can only capture outbound requests from meshed resources.

Reply:
Since traffic splitting is done on the client side, if an unmeshed resource sends traffic to the apex service of a traffic split, the traffic split logic will not be applied and the traffic will go directly to the apex service.

Reply:
@adleong Thanks for the clarification on traffic split behavior. So there is a scenario in which the apex, leaf1 and leaf2 in a traffic split could all be getting unique streams of traffic?

Reply:
Yes, there is a scenario where apex, leaf1, and leaf2 all get unique streams of traffic: any unmeshed pods sending to apex will not use the traffic split and the traffic will simply go to apex; meanwhile any mesh pods sending to apex will use the traffic split and divide that traffic between leaf1 and leaf2 according to the weights.

But I'd argue that the traffic from the unmeshed source which goes directly to apex is not part of the traffic split (the unmeshed source doesn't use the traffic split) and should not be included in the traffic split UI.  (This is one of the key distinctions between showing stats for the service and showing stats for the traffic split)

Reply:
Got it, that makes sense as we are only looking at the state of the traffic split and not the service as a whole. 👍  

Given that, what did you mean in your initial comment by, 
"...we can match exactly against the `dst` for the apex and the `dst_service` for the leaf"

Outbound requests in Prometheus can have a `dst_service` but not a plain `dst`. My plan would be to query Prometheus for outbound requests made to each leaf service matching on `dst_service`. Is this going to give us what we need?

Reply:
@scottcarol you are right, I got mixed up: there is no `dst` label at this point.  We can use the authority instead.  Since the request is going through an outbound proxy, we know the authority has been canonicalized.  Therefore, we know it will definitely be of the form: `<apex>.<apex-ns>.svc.<cluster-domain>:<port>`.